test/blackbox-tests/gen_tests.ml

```diff
@@ -178,6 +178,7 @@ let exclusions =
   (* for the following tests sandboxing is disabled because absolute paths end up
      appearing in the output if we sandbox *)
   ; make "env-bins" ~disable_sandboxing:true
+  ; make "corrections"  ~skip_platforms:[Mac] (* uses bash *)
```

Comment:
Note that you're free to use bash in tests. You just need to make sure it works with bash 3.2 (the last GPL2 version of bash). But really, it's still something to avoid if we ever want to port out test suite to BSD, windows, etc.